### PR TITLE
add version logging to cloud-controller

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:
add version to cloud-controller 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
ref #27744

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/kubeadm.go [hunk 1]
```diff
@@ -20,15 +20,19 @@ import (
 	"flag"
 	"os"
 
-	_ "github.com/golang/glog"
+	glog "github.com/golang/glog"
```
Comment (rosti) [inline cmd/kubeadm/app/kubeadm.go]: Just a tiny nit, you can drop the glog alias here and leave this line as `"github.com/golang/glog"`

**Review by rosti — APPROVED** [review]
Looks superb from kubeadm's side

File: cmd/kubeadm/app/kubeadm.go [hunk 2]
```diff
@@ -20,15 +20,19 @@ import (
...
 
 	utilflag "k8s.io/apiserver/pkg/util/flag"
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd"
+	"k8s.io/kubernetes/pkg/version"
 )
 
 // Run creates and executes new kubeadm command
 func Run() error {
+	// To help debugging, immediately log version
+	glog.Infof("Version: %+v", version.Get())
```
Comment (dixudx) [inline cmd/kubeadm/app/kubeadm.go]: Why not using `kubeadm version` to get the version?

I don't think we should add it. Neither for above `Version: xx` changes.

**Review by dixudx — CHANGES_REQUESTED** [review]

File: cmd/kubeadm/app/kubeadm.go (hunk 2 above)
Comment (tanshanshan) [reply cmd/kubeadm/app/kubeadm.go]: I remove it , THX

**Review by yastij — APPROVED** [review]
/lgtm

**Comment by nikhita** [comment]
/assign @thockin 
for approval

**Review by cheftako — APPROVED** [review]
/lgtm

**Comment by cheftako** [comment]
This helps bring this in line with the KCM which is good.

**Pull request merged**